controllers/device_controller.go

```diff
@@ -256,28 +239,26 @@ func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev
 
 // Update the actual property value of the device on edge platform,
 // return the latest status and the names of the property that failed to update
-func (r *DeviceReconciler) reconcileDeviceProperties(d *devicev1alpha1.Device, deviceStatus *devicev1alpha1.DeviceStatus, log logr.Logger) (*devicev1alpha1.DeviceStatus, []string) {
+func (r *DeviceReconciler) reconcileDeviceProperties(d *devicev1alpha1.Device, deviceStatus *devicev1alpha1.DeviceStatus) (*devicev1alpha1.DeviceStatus, []string) {
 	newDeviceStatus := deviceStatus.DeepCopy()
 	// This list is used to hold the names of properties that failed to reconcile
 	var failedPropertyNames []string
 	// 2. reconciling the device properties' value
-	log.V(3).Info("reconciling the device properties' value")
+	klog.V(3).Info("reconciling the value of device properties")
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.